### PR TITLE
feat: created home-page for AgroStores Website

### PR DESCRIPTION
**Pull request by kedark152**
### feat: created home-page for AgroStores Website.
1. Navbar for e-commerce website.
2. Header with Title, Image and Shop now button.
3. Categories section with 4 category items.
4. footer

Preview Link: https://deploy-preview-1--agrostores.netlify.app/

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+.title-underline {
+    background-color: var(--primary);
+    width: 10rem;
+    height: 0.25rem;
+    margin: 0.5rem auto 2rem;
+}
+.categories{
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(5rem, 15rem));
```
Comment (lannisterrr) [inline styles.css:52]: just a suggesstion , you can also use flex and give flex-wrap for responsiveness, as grid is mostly used for the overall layout of the page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        <img
+          class="img-responsive"
+          src="./assets/Images/pesticides.jpg"
+          alt=""
+        />
+      </div>
+      <div class="category-items cat-item-3">
+        <a href="#">
+          <p class="cat-item-text para-lg fw-bold">Tools</p>
+          <img class="img-responsive" src="./assets/Images/tools.jpg" alt="" />
```
Comment (vivekmehto) [inline index.html]: always mention the alt attribute of the image.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        </a>
+      </div>
+      <div class="category-items cat-item-4">
+        <a href="#">
+          <p class="cat-item-text para-lg fw-bold">Seeds</p>
+          <img class="img-responsive" src="./assets/Images/seeds.jpg" alt="" />
```
Comment (vivekmehto) [inline index.html]: here also.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        One Stop Solution for your farming needs.
+      </h1>
+      <h2 class="banner-subtitle lt-sp-1 text-center">
+        Buy Fertilizers, Insectisides, Seeds & other Agriculture Products
+      </h2>
+      <a href="" class="btn btn-solid btn-shop-now">Shop Now</a>
+    </header>
+    <!-- Categories -->
+    <h2 class="text-center mg-top-md">Featured Categories</h2>
+    <div class="title-underline"></div>
```
Comment (vivekmehto) [inline index.html:72]: I think this div is not used you can remove it.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    border: 2px solid var(--white);
+}
+.title-underline {
+    background-color: var(--primary);
+    width: 10rem;
+    height: 0.25rem;
+    margin: 0.5rem auto 2rem;
+}
+.categories{
+    display: grid;
```
Comment (vivekmehto) [inline styles.css:51]: Just a suggestion if we use flex here and justify-content center it would look really good.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    height: 80vh;
+    color: var(--secondary);
+    background-image: url(./assets/Images/farm.jpg);
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position: center;
+    position: relative;
+}
+
+.banner-title{
```
Comment (vivekmehto) [inline styles.css:21]: just a suggestion that if that banner title and sub-title is in the middle it would look really good.

**Review by vivekmehto — COMMENTED** [review]
Few suggestions for the alt attribute and code look clean and the color scheme is really good.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    <div class="categories">
+      <div class="category-items cat-item-1">
```
Comment (ShreyashMahajan) [inline index.html]: better naming can be used instead of 'cat' here

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+.btn-shop-now{
+    position: absolute;
+    background-color: white;
+    border: 2px solid var(--primary-hover);
+    color: var(--primary);
+    width: 10rem;
+    font-size: 1.5rem;
+    padding:0.5rem 1.5rem;
```
Comment (ShreyashMahajan) [inline styles.css:32]: use px for less than 1rem

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+.btn-shop-now:hover{
+    color: white;
+    border: 2px solid var(--white);
+}
+.title-underline {
+    background-color: var(--primary);
+    width: 10rem;
+    height: 0.25rem;
```
Comment (ShreyashMahajan) [inline styles.css]: same here

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+  border-radius: 5px;
+  
+}
+
+.cat-item-text{
+    position: absolute;
+    top: 4rem;
+    left:4rem;
+    color: var(--secondary);
+    background-color: rgba(255, 255, 255, 0.432);
```
Comment (ShreyashMahajan) [inline styles.css]: you can use variable for colors

**Review by ShreyashMahajan — APPROVED** [review]
UI looks amazing, Code structure and naming are also good. One suggestion that Use images of less size, like the size of the farm image is very big (4Mb). You can use webp images also

File: styles.css (hunk 1 above)
Comment (kedark152) [reply styles.css:52]: Thanks for your Review, I am planning to add some more categories in future so thought of grid will be best to use.

File: index.html (hunk 2 above)
Comment (kedark152) [reply index.html]: Thanks for your Review, I have made the changes!!

File: index.html (hunk 3 above)
Comment (kedark152) [reply index.html]: Thanks for your Review, I have made the changes!!

File: index.html (hunk 4 above)
Comment (kedark152) [reply index.html:72]: This div is used for bold line.

File: styles.css (hunk 5 above)
Comment (kedark152) [reply styles.css:51]: Thanks for your Review, I am planning to add some more categories in future so thought of grid will be best to use.

File: styles.css (hunk 6 above)
Comment (kedark152) [reply styles.css:21]: Thanks Vivek.

File: index.html (hunk 7 above)
Comment (kedark152) [reply index.html]: Thanks! I have updated.

File: styles.css (hunk 9 above)
Comment (kedark152) [reply styles.css]: Thanks Updated

File: styles.css (hunk 10 above)
Comment (kedark152) [reply styles.css]: Thanks Updated!!

**Review by vivekmehto — APPROVED** [review]
Now looks good.

**Pull request merged**